### PR TITLE
Add PDF Viewer

### PR DESCRIPTION
**Pull request by williamputraintan**
- PDF viewer is working with `<iframe>` tag
- Add "Open In New Tab" on the data action button

Screenshot 2023-05-05 at 22 34 33

### CODE REVIEW

**Comment by victorskl** [comment]
You are super quick! 

Could you also pls add in respective Trello stroy card that link to this PR.

I will review soon. Thank you!

**Comment by victorskl** [comment]
FYI @reisingerf @alexiswl @skanwal

**Comment by williamputraintan** [comment]
[Portal Release - Ocicat: Support PDF displays](https://trello.com/c/wxeDJhtr/1430-portal-release-ocicat-support-pdf-displays)

**Review by reisingerf — APPROVED** [review]
LATM

**Comment by victorskl** [comment]
> LATM

Looking-At-The-Moment? 😬

**Comment by reisingerf** [comment]
Looks Alien To Me ;-)

**Comment by victorskl** [comment]
> Looks Alien To Me ;-)

OMGosh! I like this! 😭 I'm gonna use that; seriously.

File: src/api/gds.ts [hunk 1]
```diff
@@ -54,14 +54,30 @@ export type PresignApiData = {
...
+    {
+      "headers": {
+        "Content-Disposition": "inline",
+        "Content-Type": "text/html"
+      }
+    }
+ * @param id
+ * @param apiConfig
+ * @returns
+ */
```
Comment (victorskl) [inline src/api/gds.ts:80]: Nice! thanks for documenting this, Will. 👍

**Review by victorskl — APPROVED** [review]
LGTM! Please squash merge.

**Pull request merged**